.babelrc.js

```diff
@@ -9,8 +9,7 @@ module.exports = {
       '@babel/preset-env',
       {
         modules: isEs ? false : 'commonjs',
-        loose,
-        ...(test ? { targets: { node: '8' } } : {})
```

Comment:
Just for reference: this line was also removed in #4470

Reply:
Does anyone know why the test target was configured not to match the built product? This lead to several tests passing that should have failed. I'm hesitant to delete this without understanding why it was setup this way in the first place.

Reply:
@esetnik This file was introduced here: https://github.com/erikras/redux-form/commit/7b75f1b71ace0bb4805f754347dbeb7ef96a812f#diff-8363ab4678bef8ef4d374bd410e88532 ... If you compare with older `babelrc` it looks like wasn't something expected. But, you're right, we should test using the same config from released product.